geode-protobuf/src/main/proto/clientProtocol.proto

```diff
@@ -49,9 +49,6 @@ message Request {
         RemoveRequest removeRequest = 6;
         RemoveAllRequest removeAllRequest = 7;
 
-        CreateRegionRequest createRegionRequest = 21;
-        DestroyRegionRequest destroyRegionRequest = 22;
-
 //        PingRequest pingRequest = 41;
```

Comment:
Can we remove the Ping messages as well?

Reply:
I'll remove the ping req/responses